server/src/main/java/com/tencent/rss/server/Checker.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.server;
+
+public abstract class Checker {
+
+  Checker(ShuffleServerConf conf) {
```

Comment:
The construct is not necessary for now?

Reply:
Hope checker‘s subclass's contructor has `ShuffleServerConf` parameter.

Reply:
maybe I should write as below,
public abstract class Checker {

  protected ShuffleServerConf conf;
  
  Checker(ShuffleServerConf conf) {
    this.conf = conf;
  }

  abstract boolean checkIsHealthy();
}